Issue 11 cool weekly contributors graph

kobebuckley opened the pull request:
## Summary

The changes are the inclusion of the provided YML file for the GitHub action of creating a constantly updated Contributors json of Hawaii-Zoning-Atlas and HIERR. 

Inserts the list of all contributors on the website's About Us section. 

### Changes

- [x ] **Updated Section:** 

About section / Be part of our story

- [x ] **Added new feature:** 

Include an automatically updated list of contributors from Hawaii-Zoning-Atlas and HIERR.; then add it to the website in list format


### Related Issue

https://github.com/CodeWithAloha/CWAWebsite/issues/11

### Screenshots

Before

![before](https://github.com/CodeWithAloha/CWAWebsite/assets/42805189/f244a052-c98f-4b07-814b-478cf6033996)

After

![after](https://github.com/CodeWithAloha/CWAWebsite/assets/42805189/dfb06f91-1f5b-47bf-a156-56dc9c784b3f)

Alexandra-Haynes reviewed (changes requested):
@kobebuckley, this is a big challenge you took, I commend you for that 😆 
This is a great start, looks like you got the api working and you implemented a good structure for this feature. 

A few suggestions.

- I think we should drop the scaling up animation on hove, or at least add it individually for each contributor (right now while you hover the whole contributors section scales up a bit)

- Having all the contributors listed a great milestone, well done. I was thinking that we can improve the UI by also adding the profile picture of each contributor. Another cool-to-have add on would be to have the names also be a link to their GitHub profile. Again, this is a big challenge and we are gonna do it together, step by step. 
The scope of this section is to show gratitude for our contributors and also encourage people to join us, so we have to think about how to maximize this effect. 

- Please also look into how to add the contributors for this repo, CWAWebsite, too. 

This is a lot, but we'll get it done together! Let me know if you have questions and don't forget that you can always ask for help with anything, the team is always there at each step.
Thanks, Kobe! 🔥

kobebuckley commented:
Hi @Alexandra-Haynes ,

Thank you for the suggestions Alex; I updated the code with the suggestions you made. 

- Improved UI by adding Profile Picture as well as clickable link per username to contributors' GitHub pages (also update the nextConfig due to image requirements)
- Provided a secondary style for the username header for more effect (in the comment block below the contributors mapping)
- Deleted the hover effect

- Incorporated the CWAWebsite repo into the list of contributors.

![updated-after](https://github.com/CodeWithAloha/CWAWebsite/assets/42805189/33713fc4-08c0-4fa0-993c-dcac06992646)

Please let me know if anything else is needed

Alexandra-Haynes reviewed (approved):
@kobebuckley this is impressive! thank you for not giving up on this challenge! Now contributors get their earned recognition and our home website looks more fun and inviting! The whole team appreciates your effort! 🔥
